src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -730,6 +730,26 @@
       {{ attributes.placeholder is defined ?  attributes.placeholder : 'Choose file(s)'|trans({}, 'Admin.Actions') }}
     </label>
   </div>
+  {# is defined in ImageTypeExtension #}
+  {% if image_url is defined and image_url is not null %}
+    <div class="form-group row mt-3">
+      <div class="col-sm">
+        <figure class="figure">
+          <img src="{{ image_url }}" class="figure-img img-fluid img-thumbnail">
+        </figure>
+        {% if delete_action_route is defined and id is defined %}
```

Comment:
`id_property` is never used, maybe it's supposed to be used here?

Reply:
The **id_property** is used here https://github.com/PrestaShop/PrestaShop/pull/17842/files#diff-e560981ab87968f4ec8283bef85cef23R55

Reply:
it is used in the ImageTypeExtension. The solution came from https://symfony.com/doc/3.4/form/create_form_type_extension.html The only difference that used Hidden fields instead of Entity methods to get the values.